src/jrd/tra.cpp

```diff
@@ -143,7 +143,13 @@ void TRA_attach_request(Jrd::jrd_tra* transaction, Jrd::jrd_req* request)
 void TRA_detach_request(Jrd::jrd_req* request)
 {
 	if (!request->req_transaction)
+	{
+		fb_assert(!request->req_savepoints);
 		return;
+	}
+
+	// Release stored looper savepoints
+	Savepoint::destroy(request->req_savepoints);
```

Comment:
On 06/14/2016 01:58 PM, Dimitry Sibiryakov wrote:
> ` 	engine13.dll!Firebird::MemPool::release(void * object, bool flagDecr) Line 2125	C++`

at this frame please print *block and *pool